lazy_relay.js

```diff
@@ -213,11 +213,16 @@ function onIdentified(err) {
     }
 
     var conn = self.peer.getInConnection(true);
+    if (!conn) {
+        self.logger.warn('onIdentified called on non-existing connection', self.extendLogInfo({}));
+        self.onError(errors.NoPeerAvailable());
+        return;
+    }
+
     if (!conn.remoteName) {
         // we get the problem
         self.logger.warn('onIdentified called on unidentified connection', self.extendLogInfo({}));
-    }
-    if (conn.closing) {
+    } else if (conn.closing) {
```

Comment:
why this... also same remark, probably both of these warn log sites could benefit from `Peer#extendLogInfo`